hypervisor/include/arch/x86/cpu.h

```diff
@@ -83,13 +83,16 @@
 #define CR4_OSFXSR              (1UL<<9)	/* OS support for FXSAVE/FXRSTOR */
 #define CR4_OSXMMEXCPT          (1UL<<10)
 /* OS support for unmasked SIMD floating point exceptions */
+#define CR4_UMIP                (1UL<<11)	/* User-Mode Inst prevention */
 #define CR4_VMXE                (1UL<<13)	/* VMX enable */
 #define CR4_SMXE                (1UL<<14)	/* SMX enable */
+#define CR4_FSGSBASE            (1UL<<16)	/* RD(FS|GS|FS)BASE inst */
 #define CR4_PCIDE               (1UL<<17)	/* PCID enable */
 #define CR4_OSXSAVE             (1UL<<18)
+/* XSAVE and Processor Extended States enable bit */
 #define CR4_SMEP                (1UL<<20)
 #define CR4_SMAP                (1UL<<21)
-/* XSAVE and Processor Extended States enable bit */
+#define CR4_PKE                 (1UL<<22)	/* Protect-key-enable */
```

Comment:
<< 22U. Change the CR4 bit definition too.

Reply:
Yes. I noticed this one and planned to have separate patch for this change.
I will handle this in new push of this patch series.